conf.d/main

```diff
@@ -28,18 +35,15 @@ service mysql start
 service apache2 start
 
 # setup the database
-MYSQL_BATCH="mysql --user=root --password=$MYSQL_PASS --batch"
-MYSQL_ADMIN="mysqladmin --user=root --password=$MYSQL_PASS"
-
-$MYSQL_ADMIN create $DB_NAME
-$MYSQL_BATCH --execute "grant all privileges on $DB_NAME.* to $DB_USER@localhost identified by '$DB_PASS'; flush privileges;"
+mysqladmin create $DB_NAME
+mysql --batch --execute "grant all privileges on $DB_NAME.* to $DB_USER@localhost identified by '$DB_PASS'; flush privileges;"
 
-/var/www/concrete5/concrete/bin/concrete5 c5:install \
+turnkey-concrete5 c5:install \
     --db-server=localhost --db-username=$DB_USER --db-password=$DB_PASS \
-    --db-database=$DB_NAME --site="Turnkey Concrete5" --allow-as-root \
+    --db-database=$DB_NAME --site=TurnKey-Concrete5 --allow-as-root \
     --starting-point=elemental_full --admin-email=$ADMIN_MAIL --admin-password=$ADMIN_PASS
 
-cat >$WEBROOT/.htaccess<<EOF
+cat >> $WEBROOT/.htaccess <<EOF
```

Comment:
Is this change from overwrite to appending to the .htaccess file intentional?

Reply:
Good catch...

FWIW I actually put that in place to check whether the file was being created or overwritten. Turns out it's being created, so it currently makes no difference. But having said that, I'm not sure whether we should leave it, or change it back to what it was? Thoughts?

Reply:
I would've said fix it 6 days ago. But I see no reason to wait more than 6 days, so let's just merge it.